functions/src/cors.ts

```diff
@@ -1,5 +1,6 @@
-import * as cors from "cors";
+import cors from "cors";
 
 export const useCors = cors({
-  origin: true,
+  // Don't use CORS in testing mode
+  origin: process.env.NODE_ENV !== "test" ? true : false,
```

Comment:
from initial comment:

> Jest with ESM doesn't support mocking ESM modules so instead of mocking CORS I just disable it for testing.

Reply:
You don't need explicit true/false and `process.env.NODE_ENV !== "test"` should get you that boolean.